ci/storage/test_main_storage_prepare_node.sh

```diff
@@ -15,7 +21,7 @@ if command -v ibv_devinfo; then ibv_devinfo || true; fi
 if lspci | grep NVMe; then
   find /dev -name 'pmem*'
 
-  daos_server storage prepare --scm-only --force
+  daos_server storage prepare --scm-only "$scm_opt"
```

Comment:
Per above suggestion:

```suggestion
  daos_server storage prepare --scm-only --"${scm_opt:-force"}"
```